### PR TITLE
tweak and document default settings

### PR DESCRIPTION
**Pull request by RalfJung**
I am also fine leaving the default tunnel limit at 1024, but then we should likely give an example for the kind of machine where that makes sense.

For the `connection_rate_limit`, I think 10 is a really bad default value. It means that even with just 100 clients, after a broker restart it takes 1000 seconds for all clients to reconnect! We are using 0.1 in production here.

Fixes https://github.com/wlanslovenija/tunneldigger/issues/136
Cc @yanosz

### CODE REVIEW

**Comment by RalfJung** [comment]
> ooks ok. I'd cool if you could be more specific on the VM and bare metal host
(e.g. Amazon EC2 T3 nano - year 2020, or whatever host you use).

The VMs we are using are on various European hosters, nothing standardized. I doubt they can be compared to other hosters or even within the same hoster, these offers change rapidly and they don't tell you the details of the hardware or how many customers there are per physical machine. On the plus side, they are really cheap. ;)

I don't think there is any useful way to name "reference VMs" here.

**Comment by yanosz** [comment]
Thanks for your comment and effort. Maybe other people can share their experience, too. IMHO: Having a small doc section on hardware requirements (i.e. size of running installations) appears to be realistic.

**Comment by RalfJung** [comment]
@kaechele @mitar what do you think about these changes to the default settings?

File: broker/l2tp_broker.cfg.example [hunk 1]
```diff
@@ -5,13 +5,15 @@ address=127.0.0.1
...
+; Maximum number of tunnels that will be allowed by the broker.
+; On a cheap VPS, more than 256 tunnels usually do not make sense.
+; Physical machines can take more.
+max_tunnels=256
 ; Tunnel id base
 tunnel_id_base=100
 ; Reject connections if there are less than N seconds since the last connection.
 ; Can be less than a second (e.g., 0.1).
-connection_rate_limit=10
+connection_rate_limit=0.2
```
Comment (mitar) [inline broker/l2tp_broker.cfg.example:16]: Hm, maybe we should have rate limiting per source IP, not global. Otherwise rate limiting is really hard.

**Comment by mitar** [comment]
Looks good to me.

File: broker/l2tp_broker.cfg.example (hunk 1 above)
Comment (RalfJung) [reply broker/l2tp_broker.cfg.example:16]: @mitar  You mean something like https://github.com/wlanslovenija/tunneldigger/pull/138? :D

Comment (mitar) [reply broker/l2tp_broker.cfg.example:16]: :-) So let's update then defaults then. :-)

**Pull request merged**